zulipterminal/ui_tools/buttons.py

```diff
@@ -297,11 +297,18 @@ def __init__(
             stream_name=self.stream_name,
             topic_name=self.topic_name,
         )
+
+        topic_prefix = " "
```

Comment:
Minor: I'd consider a comment here, simply since this is one space but ends up adding two due to there now being a prefix character.